conmon-rs/server/src/child_reaper.rs

```diff
@@ -341,6 +355,27 @@ impl ReapableChild {
         Ok((exit_tx, exit_rx))
     }
 
+    async fn spawn_cleanup_process(raw_cmd: &mut Vec<String>) -> Result<()> {
+        let mut cleanup_cmd = Command::new(raw_cmd.remove(0));
+
+        raw_cmd.iter().for_each(|arg| {
+            cleanup_cmd.arg(arg);
+        });
+
+        match cleanup_cmd.status().await {
```

Comment:
This is almost there. need a tokio::spawn here to move the status await into a green thread, so it doesn't block the function.

Reply:
ah ok, I'll get on that right now

Reply:
should be set now, let me know if there's anything else!